lib/screens/create_game1/create_game_screen1.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:niira/navigation/navigation.dart';
+import 'package:niira/screens/create_game1/game_name_field.dart';
+import 'package:niira/screens/create_game1/password_field.dart';
+import 'package:niira/screens/create_game2.dart';
+import 'package:niira/services/game_service.dart';
+import 'package:provider/provider.dart';
+
+class CreateGameScreen1 extends StatefulWidget {
+  static const routeName = '/create_game1';
+  const CreateGameScreen1({Key key}) : super(key: key);
+  @override
+  _CreateGameScreen1State createState() => _CreateGameScreen1State();
+}
+
+class _CreateGameScreen1State extends State<CreateGameScreen1> {
+  final _formKey = GlobalKey<FormState>();
+  bool _autoValidateForm = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: true,
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+        centerTitle: false,
+        title: const Text('Create Game 1/2',
+            style: TextStyle(color: Colors.white)),
+        actions: [
+          FlatButton.icon(
```

Comment:
new code

Reply:
Love this style of pointing out where to look.  We're gonna be breezing through pr's soon!